content/en/docs/installation/compatibility.md

```diff
@@ -43,6 +43,10 @@ deployment, or, if using Helm, configuring it in your `values.yaml` file.
 Note that since kubelet uses port `10250` by default on the host network, the
 `webhook.securePort` value must be changed to a different, free port.
 
+Cert-manager is not compatible with AWS Fargate, it must be run inside a 
+nodegroup. For more information see 
```

Comment:
must be run feels weird to me... could just be me :)

Reply:
open to corrections :)

Reply:
How about

```
cert-manager is not compatible with AWS Fargate, in order to use it it must be placed in a nodegroup
```

Hope that sounds better? However never having tried fargate i have no idea if this is correct :)

Reply:
accepted.
* re-based against `master`
* squashed commits
* added `Signed-off-by`

Fargate is pretty sick, IMO its the future of Kubernetes on AWS. Its a fully managed service with no EC2 VMs exposed to users so there's less things to worry about or tinker with. The downside of this is that Kubernetes `Nodeport` and `DaemonSet` simply don't exist on this platform, so systems must be implemented to take account of this. :sunglasses:

Reply:
I never liked EKS personally it was too much work t be on a cloud... Thanks for the help!